### PR TITLE
Change record page layout

### PR DESCRIPTION
**Pull request by ConnorSheremeta**
- Changed record page layout to reflect mock-up
- Added font-awesome gem
- Removed remaining glyphicons assets (removed from bootstrap in version 4)
#1426

### CODE REVIEW

**Comment by ConnorSheremeta** [comment]
Note: this is based off of my upgrade_bootstrap branch due to it being a dependency of this change. Until that is merged, this PR will include all changes made from both branches (which will make it difficult to review.)

**Comment by ConnorSheremeta** [comment]
Changes to record page layout are now live at http://uatsrv01.library.ualberta.ca:3012

I wanted to confirm in particular that the holdings table had proper layout and labels as pictured here @seanluyk :

1678SS

**Comment by theLinkResolver** [comment]
@seanluyk No item type in the new display? Is that on purpose? I recommend caution as our item types include circulation policies such as Local Loan (this is terrible practice but it is what it is, without institutional and consortial investment in changing it).

**Comment by seanluyk** [comment]
@theLinkResolver good question re: item type. If you take a look at NEOS discovery, item type is not included in the holdings table. I imagine that this was a conscious decision to simplify it, but we can always look at this again. The record view change is modelled on NEOS discovery, so that's why you aren't seeing it here. That being said, @ConnorSheremeta would you be able to add in item type below status for symphony records so we could see what it looks like?

**Comment by seanluyk** [comment]
@ConnorSheremeta I'm really happy with the new and improved record layout, thanks so much for your work on this! In addition to adding in item type, I have a few other minor comments:
- In the tools menu, if it isn't too difficult, could we change the order of links to match NEOS Discovery (just for consistency): 
image
- For the Kule holdings table, could the location be placed at the top of the box like in the SFX and Symphony tables, and the access label be removed?
 - Just an FYI that in our usage rights in SFX records, this is soon to be replaced (see #1315)

Other than that I think this is good to go, and I can document any other improvements in separate issues

**Comment by ConnorSheremeta** [comment]
Addressed your comments @seanluyk , attached new screenshots below:
Screenshot from 2019-06-24 15-50-52
___
Screenshot from 2019-06-24 15-08-55
___
Screenshot from 2019-06-24 15-01-46

**Comment by seanluyk** [comment]
Looking good, thanks for making these changes @ConnorSheremeta! After talking with @weiweishi today, we think that it would be a good idea for @nnunn to review this PR so we have another set of (UX focused) eyes on it

**Comment by weiweishi** [comment]
@ConnorSheremeta @seanluyk I have tagged @nnunn as a reviewer for the PR. The code looks good to me.

**Comment by ConnorSheremeta** [comment]
Added fix for #1488, once this is merged I'll close that issue

File: app/views/catalog/_holdings.html.erb [hunk 1]
```diff
@@ -1,72 +1,70 @@
...
+                  <% else %>
+                    Unknown/Never
+                  <% end %>
+                <% else %>
+                  <%= symphony_status(item) %>
+                  <% if @unavailable %>
+                    <p style="font-size: small">
+                      <strong>
+                        Note: For access, please contact any University of Alberta Libraries service desk in person or at
+                        <a href="http://www.library.ualberta.ca/askus" target="_blank">
```
Comment (pgwillia) [inline app/views/catalog/_holdings.html.erb]:
```suggestion
                        <%= link_to 'Ask Us', cms_root_url('ask-us'), target: '_blank' %>
```

File: app/views/catalog/_holdings.html.erb [hunk 2]
```diff
@@ -1,72 +1,70 @@
...
+                          www.library.ualberta.ca/askus.
```
Comment (pgwillia) [inline app/views/catalog/_holdings.html.erb]:
```suggestion
                         <%= cms_root_url('ask-us') %>
```

File: app/views/catalog/_holdings.html.erb [hunk 3]
```diff
@@ -91,34 +89,50 @@
...
         <tr>
-          <th>Subscription</th>
-          <th>Coverage</th>
-          <th>Note</th>
-          <th>Our Usage Rights</th>
+          <td>
+            <!--Subscription-->
+            <span class="where-is-this-item">
+              <strong><%= item[1][:name] %></strong>
+              <a href="<%= item[1][:url] %>" target='_blank'><%= icon('fas', 'info-circle') %></a>
```
Comment (pgwillia) [inline app/views/catalog/_holdings.html.erb]: Opportunity to use link_to here.

File: app/views/catalog/_holdings.html.erb [hunk 4]
```diff
@@ -130,30 +144,34 @@
...
-            <% if @document.respond_to?( :to_email_text ) %>
-              <%= link_to "Email Me", email_catalog_path(:id => @document, :call=>item[:callnumber], :location=>item[:location]) , {:id => 'emailLink', :class => 'lightboxLink'} %>
-            <% end %>
-            <% if @document.respond_to?( :to_sms_text ) %>
-              <%= link_to "Text Me", sms_catalog_path(:id => @document, :call=>item[:callnumber]), {:id => 'smsLink', :class => 'lightboxLink'} %>
-            <% end %>
+            <!--Access-->
+            <span class="where-is-this-item">
+              <strong><%= item[:access_title] %></strong>
+              <a href="<%= item[:access_url] %>" target="_blank"><%= icon('fas', 'info-circle') %></a>
```
Comment (pgwillia) [inline app/views/catalog/_holdings.html.erb]: Another opportunity to use link_to

File: app/views/catalog/_show_sidebar.html.erb [hunk 5]
```diff
@@ -6,55 +6,59 @@
...
-        </li>
-      <% end %>
+  <%# bookmark functions for items/docs -%>
+  <%= render_index_doc_actions @document, wrapping_class: 'index-document-functions col-sm-3 col-lg-2' %>
 
-      <li class= "nav-item">
-        <%= link_to 'Librarian View', librarian_view_catalog_path(id: @document), id: 'citeLink', class: 'nav-link lightboxLink' %>
-      </li>
+  <ul class="nav nav-stacked">
+    <li class= "nav-item"><a href="/bookmarks">Go to Bookmarks</a></li>
```
Comment (pgwillia) [inline app/views/catalog/_show_sidebar.html.erb]: Another opportunity to use link_to

File: app/views/catalog/_show_sidebar.html.erb [hunk 6]
```diff
@@ -6,55 +6,59 @@
...
-      <% end %>
-    </ul>
-    <%# bookmark functions for items/docs -%>
-    <%= render_index_doc_actions @document, wrapping_class: 'index-document-functions col-sm-3 col-lg-2' %>
-  </div>
+    <li class="nav-item send_correction">
+      <%= link_to 'Send Correction', new_catalog_correction_path(catalog_id: @document['id']), class: 'nav-link' %>
+    </li>
+
+    <li class= "nav-item"><a href="/search_history">View Search History</a></li>
```
Comment (pgwillia) [inline app/views/catalog/_show_sidebar.html.erb]: Another opportunity to use link_to

File: spec/features/catalog_search_spec.rb [hunk 7]
```diff
@@ -21,8 +21,8 @@
     VCR.use_cassette('sfx_result') do
       visit 'catalog/954921333008'
       expect(page).to have_text('Accountancy') # title
-      expect(page).to have_link('Business Source Complete') # Subscription
-      expect(page).to have_link('Factiva') # Subscription
+      expect(page).to have_text('Business Source Complete') # Subscription
```
Comment (pgwillia) [inline spec/features/catalog_search_spec.rb]: This change brought something to my attention
Screenshot from 2019-07-05 11-42-15
Screenshot from 2019-07-05 11-39-01
The link to 'Factiva' is now more hidden than before.  (Hint: it's under the information icon).  Is there another way to present this?

**Comment by ConnorSheremeta** [comment]
I was modelling the mobile page after NEOS discovery (which has the left-sidebar first on mobile), it does make more sense though to have it after the main content which is now how it is displayed (screenshot below.) I've changed the links to use the link_to helper and I've changed the SFX subscription link back to being the text itself rather than under an information icon, which I think makes more sense (screenshot below.)

Screenshot from 2019-07-08 12-22-34

**Comment by seanluyk** [comment]
@ConnorSheremeta this is a great improvement, thanks!

**Comment by weiweishi** [comment]
This looks good to me based on the comments from @pgwillia and changes @ConnorSheremeta has made.

**Pull request merged**